com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/antlr/LuaLexer.java

```diff
@@ -12,7 +12,7 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-package com.dynamo.bob.pipeline.luaparser;
+package com.dynamo.bob.pipeline.antlr;
```

Comment:
Can we not use the external package/jar that we're using? (see comment above)

Reply:
No. See comment above.